cms/djangoapps/contentstore/tests/tests.py

```diff
@@ -341,11 +341,11 @@ def test_course_key_decorator(self, course_key, status_code):
         """
         Tests for the ensure_valid_course_key decorator.
         """
-        url = '/import/{course_key}'.format(course_key=course_key)
+        url = '/export/{course_key}'.format(course_key=course_key)
```

Comment:
Good question - I'm not sure why this was changed, so I changed it back. Though, whether it's import or export in this particular case shouldn't matter; they should both 404 if the course doesn't exist.

I also just realized that the reason this test was failing on Jenkins was due to a URL being wrong (silly me). I was accidentally not including this file while testing locally.